cscs-checks/libraries/math/dgemm.py

```diff
@@ -16,23 +16,21 @@ def __init__(self):
 
         self.valid_systems = ['daint:gpu', 'daint:mc', 'dom:gpu', 'dom:mc']
         self.valid_prog_environs = ['PrgEnv-gnu']
-
         self.num_tasks = 0
         self.num_tasks_per_node = 1
         self.num_tasks_per_core = 1
         self.num_tasks_per_socket = 1
         self.use_multithreading = False
-
         self.build_system = 'SingleSource'
         self.build_system.cflags = ['-O3', '-fopenmp']
-
+        perf_units = 'GFlops/sec'
```

Comment:
This is too much! Just put `Gflop/s` everywhere! The reader should not look around for the units... You'd have saved typing as well!